src/openrct2/CmdlineSprite.cpp

```diff
@@ -292,15 +293,15 @@ int32_t cmdline_for_sprite(const char** argv, int32_t argc)
         else if (argc == 2)
         {
             const char* spriteFilePath = argv[1];
-
-            if (!sprite_file_open(spriteFilePath))
+            auto sFile = sprite_file_open(spriteFilePath);
```

Comment:
(This is on commmit https://github.com/OpenRCT2/OpenRCT2/pull/13590/commits/030ba011fa142136fd7ba7a9538911c9679a5eec)

I'm not sure I'd have changed `spriteFile` to ` sFile`, not sure it improved.

Reply:
It was due to wanting to avoid a name clash whilst half way through the changes